app/src/main/res/values/colors.xml

```diff
@@ -27,7 +27,7 @@
     <color name="secondaryLightColor">#8C8C8C</color>
     <color name="secondaryDarkColor">#4A4A4A</color>
 
-
+    <color name="undoneHabitRed">#E88686</color>
```

Comment:
Do we need this now that we don't have a done checkmark?

Reply:
I meant to add a colour for the Done/Undone but apparently we don't need it.